### PR TITLE
Remove repoman commit

### PR DESCRIPTION
**Pull request by ceamac**
There are 2 commits, only the first is actually needed, the second is nice to have.

### CODE REVIEW

File: templates/commit-snippet-2 [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Code for checking consistency of @@EBUILD@@
 pushd @@CP@@ > /dev/null
 
-repoman -d full || exit 1
-repoman -m "@@CP@@: ${DESCR}" commit || exit 1
+pkgcheck scan || exit 1
```
Comment (ceamac) [inline templates/commit-snippet-2:4]: Not strictly needed, `pkgdev commit` does a scan anyway

File: templates/updatebug [hunk 2]
```diff
@@ -37,18 +37,15 @@ def main():
...
+	comment = "@@ARCH@@ done"
 	params['cc'] = {}
 	if has_my_arch:
 		params['cc']['remove'] = ['@@ARCH@@@gentoo.org']
 	params['comment'] = {}
 	if has_other_arches or 'Security' in response['product']:
 		params['comment']['body'] = comment
 	else:
-		params['comment']['body'] = comment + ', closing'
+		params['comment']['body'] = comment + '\r\n\r\nall arches done'
```
Comment (ceamac) [inline templates/updatebug:48]: Not sure this is right

File: templates/commit-snippet-2 (hunk 1 above)
Comment (thesamesam) [reply templates/commit-snippet-2:4]: `pkgdev commit` only scans when configured to, so this is good/needed

File: templates/commit-snippet-2 [hunk 3]
```diff
@@ -1,7 +1,8 @@
...
+git add @@EBUILD@@
```
Comment (DerDakon) [inline templates/commit-snippet-2:5]: Why isn't an error here fatal also? In the end we should remove all these ```|| exit 1``` and just add ```set -e ``` at the beginning of the script.

**Comment by thesamesam** [comment]
Thank you!

**Pull request closed without merging**